src/index.html

```diff
@@ -4,9 +4,387 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="header__logo"
+          class="header__logo-image"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">Apple</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">Samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#smatphones" class="header__nav-link">Smartphones</a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#laptops&computers"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#gadgets" class="header__nav-link">Gadgets</a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add a blank line between the links for better readability
```suggestion
          </li>
          
          <li class="header__nav-item">
```